Some instances of AC_ARG_ENABLE calls in configure.ac overwrite the
enable_* configure variable with 'enable_*=yes' in the third parameter.
This has the effect of turning a '--disable-*' config option into a
'--enable-*' option.  This change makes '--disable-*' work correctly.

Upstream-Status: Pending

Signed-off-by: Donn Seeley <donn.seeley@windriver.com>

---
 configure.ac |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/configure.ac
+++ b/configure.ac
@@ -129,7 +129,7 @@ AM_CONDITIONAL([ENABLE_EXAMPLES], [test
 # Python3 module and scripts
 AC_ARG_ENABLE([python],
 	[AC_HELP_STRING([--enable-python], [enable python binding])],
-	[enable_python=yes], [enable_python=no])
+	[], [enable_python=no])
 AM_CONDITIONAL([ENABLE_PYTHON], [test "x$enable_python" = "xyes"])
 
 AM_COND_IF([ENABLE_PYTHON],
@@ -146,7 +146,7 @@ fi
 
 AC_ARG_ENABLE([lua],
 	[AC_HELP_STRING([--enable-lua], [enable lua binding])],
-	[enable_lua=yes], [enable_lua=no])
+	[], [enable_lua=no])
 
 AM_CONDITIONAL([ENABLE_LUA], [test "x$enable_lua" = "xyes"])
 
@@ -157,7 +157,7 @@ AM_COND_IF([ENABLE_LUA],
 # Optional test binaries
 AC_ARG_ENABLE([tests],
 	[AC_HELP_STRING([--enable-tests], [build test/example binaries])],
-	[enable_tests=yes], [enable_tests=no])
+	[], [enable_tests=no])
 AM_CONDITIONAL([ENABLE_TESTS], [test "x$enable_tests" = "xyes"])
 
 # LXC container path, where the containers are actually stored
@@ -187,7 +187,7 @@ AC_ARG_WITH([rootfs-path],
 AC_MSG_CHECKING([Whether to place logfiles in container config path])
 AC_ARG_ENABLE([configpath-log],
 	[AC_HELP_STRING([--enable-configpath-log], [use logfiles in config path])],
-	[use_configpath_logs=yes], [use_configpath_logs=no])
+	[], [use_configpath_logs=no])
 AC_MSG_RESULT([$use_configpath_logs])
 AM_CONDITIONAL([USE_CONFIGPATH_LOGS], [test "$use_configpath_logs" = "yes"])
 
